src/routes/postSkipSegments.js

```diff
@@ -219,20 +306,24 @@ module.exports = async function postSkipSegments(req, res) {
             return;
         }
 
-        let autoModerateResult = await autoModerateSubmission({videoID, startTime, endTime});
-        if (autoModerateResult) {
-            res.status(403).send("Request rejected by auto moderator: " + autoModerateResult);
-            return;
+        // Auto moderator check
+        if (!isVIP) {
+            let autoModerateResult = await autoModerateSubmission({userID, videoID, startTime, endTime, category: segments[i].category});
+            if (autoModerateResult == "NB disagreement."){
+                // If NB automod rejects, the submission will start with -2 votes
+                decreaseVotes = -2;
```

Comment:
For now, let's set it to `0`, but we can keep the code in for when we are ready to reject by default.